README.md

```diff
@@ -2,8 +2,8 @@
 
 A suite to handle single-cell electrophysiological data and to build and validate detailed electrical models.
 
-- [eFEL](#efel) — Electrophys Feature Extraction Library
-- [BluePyEfe](#bluepyefe) — Blue Brain Python E-feature extraction
+- [eFEL](#efel) — Electrophys Feature (E-Feature) Extraction Library
+- [BluePyEfe](#bluepyefe) — E-Feature Extraction from batches of recordings
```

Comment:
Maybe keep the `Blue Brain Python` part so that it matches the name 'BluePy' part of BluePyEfe

Reply:
Updated.